builtin/aws/ecs/main.go

```diff
@@ -4,7 +4,7 @@ import (
 	sdk "github.com/hashicorp/waypoint-plugin-sdk"
 )
 
-//go:generate protoc -I ../../../.. -I ../../../thirdparty/proto --go_opt=plugins=grpc --go_out=../../../.. waypoint/builtin/aws/ecs/plugin.proto
+//go:generate protoc -I ../../../.. -I ../../../thirdparty/proto --go_out=../../../.. --go-grpc_out=../../../.. waypoint/builtin/aws/ecs/plugin.proto
```

Comment:
If I remember right, from this PR https://github.com/hashicorp/waypoint/pull/3366, I think we need an additional flag for allowing relative paths to work properly in certain cases? Maybe @krantzinator has more insight into this one.

Reply:
I looked into that `source-relative` flag - it looks like it controls where the generated code goes (docs [here](https://developers.google.com/protocol-buffers/docs/reference/go-generated#:~:text=If%20the%20paths%3Dsource_relative%20flag%20is%20specified%2C%20the%20output%20file%20is%20placed%20in%20the%20same%20relative%20directory%20as%20the%20input%20file.%20For%20example%2C%20an%20input%20file%20protos/buzz.proto%20results%20in%20an%20output%20file%20at%20protos/buzz.pb.go.)).

Our generate code is going to the right place now, so I thin it's file! I also tried adding `--go-grpc_opt=paths=source_relative` to a random plugin (k8s apply) and re-ran generation, and there was no effect.